fuel/app/classes/ltilaunch.php

```diff
@@ -77,6 +76,23 @@ public static function config()
 		return static::$config;
 	}
 
+	public static function config_from_key($key)
```

Comment:
Needed for cases where the request isn't a regular lti launch - for instance, when the frontend loads a signed request from the backend